.travis.yml

```diff
@@ -3,16 +3,17 @@ sudo: required
 dist: trusty
 env:
   matrix:
-    - DIST=precise
+    - DIST=xenial
     - DIST=trusty
 before_install:
   - sudo apt-get -y install wget software-properties-common
-  - sudo add-apt-repository -y "deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main 9.6" && wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
-  - sudo apt-get update && sudo apt-get -y install postgresql-server-dev-all postgresql-server-dev-9.6 devscripts rsync build-essential dh-make libkrb5-dev
-  - echo '9.1\n9.2\n9.3\n9.4\n9.5\n9.6' > ~/.pg_supported_versions
+  - sudo add-apt-repository -y "deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main" && wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
```

Comment:
I think `trusty` needs to be replaced with `${DIST}`

Reply:
So! Pg extension builds are different than some of the other pbuilder based builds we use. In this case, they are built once in the travis vm for all versions of postgres, instead of in a distro chroot with a very particular dependency tree. In this case, we are building the extension for `all` distros of ubuntu and uploading it to all applicable distro repos. 

Technically, I could copy the extensions from trusty to xenial directly, but this repo is in sore need of making sure these commands work, just in case, and we at least learned that it is not needed to build 9.5 and 9.6 packages.